### PR TITLE
Add Enlightn

### LINKED ISSUE #1592: Add Enlightn to build and deploy steps
Makes Laravel-specific recommendations https://www.laravel-enlightn.com/

Best to run during CI to catch static issues and at the end of deployments to catch production configuration issues.

### PR DESCRIPTION
**Pull request by kberzinch**
Closes #1592

### CODE REVIEW

File: app/Http/Controllers/AttendanceReportController.php [hunk 1]
```diff
@@ -1,36 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace App\Http\Controllers;
-
-class AttendanceReportController extends Controller
```
Comment (JoshOldenburg) [inline app/Http/Controllers/AttendanceReportController.php:7]: This was not used anywhere anymore?

File: config/enlightn.php [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+    |
+    | Specify the analyzer classes that you wish to exclude from reporting. This
+    | means that if any of these analyzers fail, they will not be counted
+    | towards the exit status of the Enlightn command. This is useful
+    | if you wish to run the command in your CI/CD pipeline.
+    | Example: [\Enlightn\Enlightn\Analyzers\Security\XSSAnalyzer::class].
+    |
+    */
+    // @phan-suppress-next-line PhanPluginDuplicateConditionalUnnecessary
+    'dont_report' => true === env('SKIP_HTTP_CHECKS', false) ? [
```
Comment (JoshOldenburg) [inline config/enlightn.php:110]: Can this just be made unconditional?

File: config/enlightn.php [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+    // Format: "myorg/myrepo" like "laravel/framework".
+    'github_repo' => env('ENLIGHTN_GITHUB_REPO'),
+
+    // Set to true to restrict the max number of files displayed in the enlightn
+    // command for each check. Set to false to display all files.
+    'compact_lines' => false,
+
+    // List your commercial packages (licensed by you) below, so that they are not
+    // flagged by the License Analyzer.
+    'commercial_packages' => [
```
Comment (JoshOldenburg) [inline config/enlightn.php:181]: Nova doesn't need to be here?

**Review by JoshOldenburg — APPROVED** [review]
Looks good. I have a few small questions that I assume are fine.

File: app/Http/Controllers/AttendanceReportController.php (hunk 1 above)
Comment (kberzinch) [reply app/Http/Controllers/AttendanceReportController.php:7]: Not that I can tell.

File: config/enlightn.php (hunk 2 above)
Comment (kberzinch) [reply config/enlightn.php:110]: I will be coming back to tweak this later. I need to do a full round of deployments so the nginx config updates are applied, then I can enable that check by default.

File: config/enlightn.php (hunk 3 above)
Comment (kberzinch) [reply config/enlightn.php:181]: Apparently not.

**Pull request merged**